ecmascript/ast/src/pat.rs

```diff
@@ -63,12 +63,12 @@ pub struct AssignPat {
 /// EsTree `RestElement`
 #[ast_node("RestElement")]
 pub struct RestPat {
+    pub span: Span,
```

Comment:
Added this because the "hi" position depends on the existence of a type annotation.